runtime/bcutil/cfreader.c

```diff
@@ -3601,3 +3606,19 @@ sortMethodIndex(J9CfrConstantPoolInfo* constantPool, J9CfrMethod *list, IDATA st
 		}
 	}
 }
+
+#if JAVA_SPEC_VERSION >= 15
+I_32
+checkClassBytes(J9VMThread *currentThread, U_8* classBytes, UDATA classBytesLength, U_8* segment, U_32 segmentLength) 
+{
+	I_32 rc = 0;
+	U_32 cfrFlags = BCT_JavaMaxMajorVersionShifted | BCT_AnyPreviewVersion | BCT_BasicCheckOnly;
+	PORT_ACCESS_FROM_VMC(currentThread);
+	if (NULL == classBytes) {
```

Comment:
Please call readClassFileBytes inside if !NULL to avoid the double return.

Reply:
Fixed.